### DISCUSSION TITLE
Modoboa Out of memory: Killed process

### DISCUSSION THREAD

**Discussion by draconMo** [modoboa/modoboa#3326] [Help]
# Impacted versions

* OS Type: Debian
* OS Version: 12
* Database Type: PostgreSQL 
* Database version: 15
* Modoboa: 2.3.2
* installer used: Yes
* Webserver: Nginx

vCPU - 3
Ram-2gb

Hello. I encountered the following problem - the RAM was 100% used.
In the logs

root@andreydracon:~# journalctl | grep "Out of"

Oct 22 00:27:58 andreydracon.ru kernel: Out of memory: Killed process 449360 (python) total-vm:197808kB, anon-rss:80352kB, file-rss:0kB, shmem-rss:0kB, UID:0 pgtables:368kB oom_score_adj:0
Oct 22 00:27:58 andreydracon.ru kernel: Out of memory: Killed process 449361 (python) total-vm:202056kB, anon-rss:85128kB, file-rss:0kB, shmem-rss:0kB, UID:0 pgtables:376kB oom_score_adj:0
Oct 22 00:27:58 andreydracon.ru kernel: Out of memory: Killed process 449479 (python) total-vm:224344kB, anon-rss:90404kB, file-rss:0kB, shmem-rss:0kB, UID:0 pgtables:400kB oom_score_adj:0

Do I understand correctly - postgresql fully used all available RAM ?

**Comment by tonioo** [comment]
@draconMo Postgres is not the issue. It's probably coming from the AV scanner (ClamAV) and in that case, you do not have enough memory available.
The documentation mentions that at least 4GB is required when running the AV scanner.
https://modoboa.readthedocs.io/en/latest/installation.html
